ask21-set.xml

```diff
@@ -504,13 +524,22 @@ See AUTHORS.txt for credits
                     <min>0</min>
                 </binding>
             </key>
+            <key n="83">
+                <name>S</name>
```

Comment:
The 'S' key is already used to increment the variomenter volume (s/S turns it up and down). Please choose another key (maybe ctrl-s?). The smoke looks great btw.